tests/phpunit/BltProject/ToggleModulesTest.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+namespace Acquia\Blt\Tests\BltProject;
+
+use Acquia\Blt\Tests\BltProjectTestBase;
+
+/**
+ * Class ToggleModulesTest.
+ *
+ * Verifies that setup:toggle-modules behaves as expected.
+ */
+class ToggleModulesTest extends BltProjectTestBase {
+
+  /**
+   * Verifies the modules for a given environment were enabled as expected.
+   *
+   * In the event no environment is specified, this test will be skipped.
+   *
+   * @group blt-project
+   */
+  public function testModulesEnabled() {
+    if (defined('BLT_ENV')) {
```

Comment:
Only if the proper variables aren't set via `phpunit.xml`, at which point the tests will be skipped but won't produce an outright failure